marker/providers/pdf.py

```diff
@@ -25,6 +27,8 @@ class PdfProvider(BaseProvider):
     ocr_space_threshold: float = .7
     ocr_newline_threshold: float = .6
     ocr_alphanum_threshold: float = .3
+    image_threshold: float = .65
+    strip_existing_ocr: bool = False
```

Comment:
This should be a cli option (in the configparser)

Reply:
sure thing!